### PR TITLE
more DB maintenance commands

### PR DESCRIPTION
**Pull request by maxtaco**
- get, put, delete
- also fix up debugging headaches

### CODE REVIEW

File: go/client/cmd_db.go [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+func (c *CmdDbNuke) ParseArgv(ctx *cli.Context) error {
+	c.force = ctx.Bool("force")
+	return nil
+}
+
+func (c *CmdDbNuke) Run() error {
+	var err error
+	if !c.force {
+		err = GlobUI.PromptForConfirmation("Really blast away your local database?")
```
Comment (patrickxb) [inline go/client/cmd_db.go]: local cache database?  this sounds more drastic than it is.

File: go/client/cmd_db.go [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+		}
+		return cli.DbNuke(context.TODO(), 0)
+	}
+	return err
+}
+
+func NewCmdDbNuke(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:  "nuke",
+		Usage: "Delete the local database",
```
Comment (patrickxb) [inline go/client/cmd_db.go]: local cache database

File: go/client/cmd_db.go (hunk 1 above)
Comment (maxtaco) [inline go/client/cmd_db.go]: this was always there.  it's showing up in the diff since i renamed the file.

Comment (patrickxb) [inline go/client/cmd_db.go]: yeah, i know, just figured while you were here....

**Pull request merged**